packages/flutter_reactive_ble/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 5.2.0
+
+* Bumps to Dart 2.17 minimum and upgrades melos to 3.1.0 in #762
+* swiftlint config and inital formatting pass in #765
+* Cancel subscription when a disconnect event has been thrown in #769
+* Fix typos in #778
+* Update CI config to use Xcode 14 in #786
+* Support multiple services or characteristics with the same id in #776
+-   Breaking change: If a device has multiple characteristics with the same id, `readCharacteristic`, `writeCharacteristic` and `subscribeToCharacteristic` used to select the first of those characteristics. Now they will fail in this case. Use `resolve` or `getDiscoveredServices` instead.
```

Comment:
Make the list bullet character consistent with the other items (`*`), remove weird spaces in the beginning of the line.

Prefer to spell ID in capital letters (or in full).